docs/hosting-a-server-with-northstar/prerequisites.md

```diff
@@ -38,6 +38,15 @@ Access your router via it's web interface and port forward
 
 to your PC that you're running Northstar on.
 
+## Firewall
+
+Open windows firewall and allow the application and port
+
+* Click add rule and choose port. Then add `37015` (UDP) and `8081` (TCP)
+* Click add rule and choose application. Then add NorthstarLauncher.exe and Titanfall2.exe
+
+`If you are using Northstar version <= 1.3, add "Titanfall2-unpacked.exe" instead of "Titanfall2.exe"`
+
```

Comment:
Personally I never had to do that and never had issues hosting. I'd rather not include steps that aren't strictly necessary. Can someone confirm?

Reply:
> Personally I never had to do that and never had issues hosting. I'd rather not include steps that aren't strictly necessary. Can someone confirm?

I have tried to disable all the rules for Northstar and Titanfall applications except port forward. And I can't connect to my server

Reply:
I have done a test in different cases. And figure out that it does not need to port forward in the Windows firewall. However, it must allow the NorthstarLauncher.exe, or else it is not able to join the server